### PR TITLE
Use Unauthorized and Forbidden correctly

### PR DESCRIPTION
**Pull request by McSinyx**
Resolve GH-144 and follow up on GH-142

### CODE REVIEW

File: src/acanban/project.py [hunk 1]
```diff
@@ -119,7 +119,7 @@ async def edit(uuid: str) -> ResponseReturnValue:
 async def report(uuid: str) -> ResponseReturnValue:
     """Return the projects' report infomation and forms."""
     user = await current_user.pluck('projects', 'role')
-    project = await pluck(uuid, ('id', 'name', 'report'), user['projects'])
+    project = await pluck(uuid, ('id', 'name', 'report'), user.get('projects'))
```
Comment (Huy-Ngo) [inline src/acanban/project.py:122]: What does this change do and why is it necessary?

**Review by Huy-Ngo — APPROVED** [review]
This seems cleaner and it resolves the issue.

File: src/acanban/project.py (hunk 1 above)
Comment (McSinyx) [reply src/acanban/project.py:122]: It fixes [this previously uncaptured error](https://builds.sr.ht/~huyngo/job/399861#task-test-191).  Turns out 100% unit coverage is insufficient and parameterization mentally helps coverring more cases.

**Comment by McSinyx** [comment]
Thanks!

**Pull request merged**